web/ui/react-app/src/pages/TSDBStatus.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, Fragment } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Alert, Table } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { useFetch } from '../utils/useFetch';
+import PathPrefixProps from '../PathPrefixProps';
+
+export interface Stats {
+  Name: string;
+  Count: number;
+}
+
+export interface TSDBMap {
+  CardinalityMetricsStats: Array<Stats>;
+  CardinalityLabelStats: Array<Stats>;
+  LabelValueStats: Array<Stats>;
+  LabelValuePairsStats: Array<Stats>;
+}
+
+function createTable(title: string, valueType: string, data: Array<Stats>) {
+  return (
+    <div>
+      <h6>{title}</h6>
+      <Table bordered={true} size="sm" striped={true}>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>{valueType}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {data.map((element, i) => {
+            return (
+              <Fragment key={i}>
+                <tr>
+                  <td>{element.Name}</td>
+                  <td>{element.Count}</td>
```

Comment:
can you lowercase ```Name``` and ```Count```? Not sure but I think @hooten advice [here](https://github.com/prometheus/prometheus/pull/6281/commits/c3442a53caa2104ef867b891c702ecbeba925533#r343181972) can help with this